src/Extensions/CommentNotifier.php

```diff
@@ -48,12 +48,13 @@ public function notifyCommentRecipient($comment, $parent, $recipient)
         $subject = $parent->notificationSubject($comment, $recipient);
         $sender = $parent->notificationSender($comment, $recipient);
         $template = $parent->notificationTemplate($comment, $recipient);
+        $to = ($recipient instanceof Member) ? $recipient->Email : $recipient;
 
         // Validate email
         // Important in case of the owner being a default-admin or a username with no contact email
-        $to = ($recipient instanceof Member) ? $recipient->Email : $recipient;
-
-        if (!Email::is_valid_address($to)) {
+        // Assume arrays are in email => name format
+        $validateTo = is_array($to) ? $validate = array_keys($to)[0] : $to;
```

Comment:
```suggestion
        $validateTo = is_array($to) ? (array_keys($to)[0] ?? '') : $to;
```

Reply:
I agree with removing the `$validate` assignment (didn't notice that there, not sure how it crept in).... but why hide an issue with the null coallescing operator? If there's a null in there, I'd prefer as a developer for that to blow up in my face with a clear "expected string, got null" error rather than having to debug why my email address isn't valid.

Reply:
Could throw an exception rather that returning when it fails validation to make the exception more descriptive.

This may fail on prod, rather than locally, so the error will be showing in error logs. It will probably also break the user experience of submitting a commit.  Probably better that it break though and puts something in the error log because having a mis-configured admin email seems like it could cause various issues.

Reply:
Actually nvm - since this is targetting `2.3` we can't make breaking changes like throwing errors where there weren't any before. I'll just make the suggested change.

Reply:
Done